termux-exec.c

```diff
@@ -72,8 +72,11 @@ static char*const * remove_ld_preload(char*const * envp)
 	return envp;
 }
 
-int execve(const char* filename, char* const* argv, char* const* envp)
+static void update_filename_argv_envp(const char **filename_ptr, char* const **argv_ptr, char*const **envp_ptr)
```

Comment:
Why remove the space between `char*` and `const`?

`update_filename_argv_envp` is not a very good name. How about `exec_handler()`?